code-examples/ragu-python-png-output.py

```diff
@@ -105,7 +106,8 @@ def graph_timeseries_data_per_router(router_traffic):
         return
 
     for router in router_traffic:
-        data = router['attributes']['view']['network']['unit']['bps']
+        router_id = 'router-' + router['id'].split('-')[1]
```

Comment:
No clue what happened to get us into this bind, but I would really like to discourage people from parsing API IDs. We don't make any guarantees that they're going to stay in a certain form, so this code could easily break if we every changed the id layout.

My suggestion would be to just loop over the views roughly like:

```python
    for router in router_traffic:
        for router_id, view in router['attributes']['view'].items():
            data = view['unit']['bps']
            ....
```